api/src/YoutubeClient.h

```diff
@@ -28,6 +30,7 @@ class VideosResult : public APIResult {
 
   rapidjson::Document GetSchema() final;
 
+
```

Comment:
wat

Reply:
hey i tried to force push this away but it didn't work
```
diff --git a/api/src/ServicePoller.cpp b/api/src/ServicePoller.cpp
index 5320087..0d65a31 100644
--- a/api/src/ServicePoller.cpp
+++ b/api/src/ServicePoller.cpp
@@ -154,7 +154,6 @@ const Status ServicePoller::CheckYouTube(const std::string& name,
     state->live = true;
     state->thumbnail = video.GetMediumThumbnail();
     state->viewers = video.GetViewers();
-    state->nsfw = video.IsAgeRestricted();
   }

   return status;
diff --git a/api/src/ServicePoller.h b/api/src/ServicePoller.h
index 5aa2fa8..510b30c 100644
--- a/api/src/ServicePoller.h
+++ b/api/src/ServicePoller.h
@@ -20,7 +20,6 @@ struct ChannelState {
   std::string thumbnail;
   uint64_t viewers;
   bool live;
-  bool nsfw;
 };

 class ServicePoller {
diff --git a/api/src/YoutubeClient.cpp b/api/src/YoutubeClient.cpp
index 51901eb..dcaa616 100644
--- a/api/src/YoutubeClient.cpp
+++ b/api/src/YoutubeClient.cpp
@@ -29,10 +29,6 @@ std::string VideosResult::Video::GetMediumThumbnail() const {
   return json::StringRef(data_["snippet"]["thumbnails"]["medium"]["url"]);
 }

-bool VideosResult::Video::IsAgeRestricted() const {
-  return data_["contentDetails"]["contentRating"]["ytRating"] == "ytAgeRestricted";
-}
-
 rapidjson::Document VideosResult::GetSchema() {
   rapidjson::Document schema;
   schema.Parse(R"json(
@@ -94,21 +90,6 @@ rapidjson::Document VideosResult::GetSchema() {
                     }
                   },
                   "required": ["viewCount"]
-                },
-                "contentDetails: {
-                  "type": "object",
-                  "properties": {
-                    "contentRating": {
-                      "type": "object",
-                      "properties: {
-                        "ytRating": {
-                          "type": "string",
-                          "pattern": "^[0-9]+$"
-                        }
-                      }
-                    },
-                    "required": ["contentRating"]
-                  },
                 }
               },
               "required": ["snippet"]
diff --git a/api/src/YoutubeClient.h b/api/src/YoutubeClient.h
index e85fdad..5b34beb 100644
--- a/api/src/YoutubeClient.h
+++ b/api/src/YoutubeClient.h
@@ -20,8 +20,6 @@ class VideosResult : public APIResult {

     uint64_t GetViewers() const;

-    bool IsAgeRestricted() const;
-
     std::string GetMediumThumbnail() const;

    private:
```

Reply:
Do we not use a formatter? I thought it was clang-format but that returned quite a large diff prior to this PR.

Reply:
yeah, it's all clang formatted with google preset